package.json

```diff
@@ -166,7 +166,7 @@
     "listr": "^0.14.3",
     "listr-verbose-renderer": "^0.6.0",
     "prettier": "^1.19.1",
-    "spectron": "^9.0.0",
+    "spectron": "^10.0.1",
```

Comment:
can you try to also update Jest and Babel Jest libs? cc @valpinkman can help

Reply:
Will be done in the first PR too 👍